src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write your code here
+const populationElements = document.querySelectorAll('.population');
+const populations = [];
+
+populationElements.forEach((element) => {
+  const population = parseInt(element.textContent.replace(/,/g, ''), 10);
+
+  populations.push(population);
+});
+
+const totalPopulation = populations.reduce((sum, num) => sum + num, 0);
+const averagePopulation = Math.round(totalPopulation / populations.length);
+
+const formatNumber = (number) =>
+  number.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
```

Comment:
Consider using curly braces `{}` for the arrow function body to improve readability, even though it's a single expression. This can help avoid errors when adding more logic in the future.